pkg/controllers/disruption/controller.go

```diff
@@ -184,7 +185,8 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command)
 		methodLabel:            m.Type(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	logging.FromContext(ctx).Infof("disrupting via %s %s", m.Type(), cmd)
+	uid := uuid.NewUUID()
+	logging.FromContext(ctx).With("command-uid", uid).Infof("disrupting via %s %s", m.Type(), cmd)
```

Comment:
```suggestion
	logging.FromContext(ctx).With("command-id", uid).Infof("disrupting via %s %s", m.Type(), cmd)
```

nit: I would just call it `id`, that's typically more common because the fact that it's unique is implied